src/app/container/container.component.ts

```diff
@@ -1,50 +1,47 @@
-import { Component } from '@angular/core';
-import { Router } from '@angular/router';
+import {Component} from '@angular/core';
+import {Router} from '@angular/router';
```

Comment:
Let's keep the code style same as the Angular 2+ official site.  See https://angular.io/styleguide